ui/src/app/components/runtime-configuration-panel/common-button.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+import { Button } from 'app/components/buttons';
+
+interface Props {
+  label: string;
+  buttonText?: string;
+  onClick: () => void;
+  disabled: boolean;
+}
+export const CommonButton = ({
```

Comment:
I find having both a "CommonButton" and "Button" confusing. My preference would be to either:
- Simplify the `Button` Component such that this isn't needed or
- Leave this as a private component in the original file so we don't end up with two publicly-scoped Buttons

Reply:
Makes sense - I put CommonButton back into R-C-P